File: 2c/calculadora.c
#include <stdio.h>

int somar(int a, int b)
{
    return a + b;
}

int subtrair(int a, int b)
{
    return a - b;
}

int multiplicar(int a, int b)
{
    return a * b;
}

int dividir (int a, int b)
{
    return a / b;
}

int main()
{
    int escolha = 0;
    int x, y = 0;

    printf("Digite um número: ");
    scanf("%d", &x);
    printf("Digite um número: ");
    scanf("%d", &y);

    printf
    (
        "Selecione seu operador!\n"
        "1 - Somar\n"
        "2 - Subtrair\n"
        "3 - Multiplicar\n"
        "4 - Dividir\n"
    );

    scanf("%d", &escolha);

    switch (escolha)
    {
    case 1:
        int q = somar(x, y);
        printf("%d", q);
        break;

    case 2:
        int w = subtrair(x, y);
        printf("%d", w);
        break;

    case 3:
        int e = multiplicar(x, y);
        printf("%d", e);
        break;

    case 4:
        int r = subtrair(x, y);
        printf("%d", r);
        break;
    
    default:
        break;
    }

    return 0;
}

File: 4javaScript/calculadora.js
function somar(a, b) {
    return a + b;
}
function subtrair(a, b) {
    return a - b;
}
function multiplicar(a, b) {
    return a * b;
}
function dividir(a, b) {
    return a / b;
}

let a = parseInt(prompt("Numero 1: "));
let b = parseInt(prompt("Numero 2: "));

console.log('1 - Somar')
console.log('2 - Subtrair')
console.log('3 - Multiplicar')
console.log('4 - Dividir')

let opcao = parseInt(prompt("Escolha uma opção (1-4): "));

switch(opcao) {
    case 1:
        alert(`Resultado: ${somar(a, b)}`);
        break;
    case 2:
        alert(`Resultado: ${subtrair(a, b)}`);
        break;
    case 3:
        alert(`Resultado: ${multiplicar(a, b)}`);
        break;
    case 4:
        alert(`Resultado: ${dividir(a, b)}`);
        break;
    default:
        alert("Opção inválida!");
}